### PR TITLE
Fixed the value to change after scrolling is completely over

### PR DESCRIPTION
**Pull request by plusone-masaki**
fix: https://github.com/wan2land/vue-scroll-picker/issues/94

### CODE REVIEW

File: src/picker/picker.js [hunk 1]
```diff
@@ -273,13 +273,13 @@ export default {
       }
       if (this.innerIndex !== index) {
         this.innerIndex = index
-        this.innerValue = index > -1 ? this.normalizedOptions[index].value : null
-        this.$emit('input', this.innerValue)
       }
```
Comment (wan2land) [inline src/picker/picker.js]: I think you should move the entire if block.
274~278

**Comment by wan2land** [comment]
Please check and make PR again. Thank you. :-)

**Comment by plusone-masaki** [comment]
I ’ve checked it now, please give me a little time:)

**Comment by plusone-masaki** [comment]
I think the behavior is the same,but it has been fixed because it looks correct to move every if block in the construction of the code.

**Pull request merged**